tests/baselines/reference/callWithSpread2.errors.txt

```diff
@@ -36,13 +38,13 @@ tests/cases/conformance/expressions/functionCalls/callWithSpread2.ts(38,1): erro
     rest("d", ...ns)
     
     
-    // this covers the arguments case
+    // extra arguments
     normal("g", ...ns)
-    normal("h", ...mixed)
-    normal("i", ...tuple)
+    ~~~~~~~~~~~~~~~~~~
+!!! error TS2556: Expected 1 arguments, but got a minimum of 2.
     thunk(...ns)
-    thunk(...mixed)
-    thunk(...tuple)
+    ~~~~~~~~~~~~
+!!! error TS2556: Expected 0 arguments, but got a minimum of 1.

```

Comment:
This may be pedantic, but then again so is this error: I think it'd be more clear to say `Expected N arguments, but got Y or more.` when a spread is involved (where `N` is the maximum arity, and `Y` is the number of non-spread arguments).

Reply:
Hm, good idea. I'll see how that looks if I change all of them.

Reply:
Yes, it is **much** easier to read. Nice suggestion. (I feel like it's a tiiny bit less accurate, but that doesn't matter much.)